environs/config/config.go

```diff
@@ -1579,6 +1610,16 @@ global or per instance security groups.`,
 		Type:        environschema.Tstring,
 		Group:       environschema.EnvironGroup,
 	},
+	ContainerImageMetadataURLKey: {
+		Description: "The URL at which the metadata used to locate container OS image ids is located",
+		Type:        environschema.Tstring,
+		Group:       environschema.EnvironGroup,
+	},
+	ContainerImageStreamKey: {
```

Comment:
If we're assuming this has the content "released", what about upgrades?
Especially WRT 2.3.5 to 2.3.6, because the value is not going to be present.
We could 
a) be double safe and check late that "" is then interpreted as "released"
b) add an upgrade step to make sure "released" gets put into the database.
Thoughts?

Reply:
I actually added the later guard, then removed it.
I think an upgrade step is best; then someone looking at the code doesn't wonder why this is both "here" and "there", and remove one.

Reply:
Step added. I think that is everything.